clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/network/view/AzureNetworkProvider.groovy

```diff
@@ -52,6 +57,17 @@ class AzureNetworkProvider implements NetworkProvider<AzureNetwork> {
     cacheView.getAll(Keys.Namespace.AZURE_NETWORKS.ns, RelationshipCacheFilter.none()).collect(this.&fromCacheData)
   }
 
+  @RequestMapping(value = "/azure/vnet/{account}/{region}/{name:.+}", method = RequestMethod.GET)
```

Comment:
I've added his mapping for testing purposes since the other mapping only provides us with a summary of the vnet and not the full description of the vnet cached object; but it can be removed as it's not needed outside of that scope (deck does not use it at the moment) and it's easy to add it back if we'll need it later.